src/one_dragon/base/operation/application/plugin_import_service.py

```
@@ -0,0 +1,494 @@
+        Returns:
+            ImportResult: 验证结果
+        """
+        file_list = zf.namelist()
+
+        # 检查是否有 factory 文件
+        has_factory = any(name.endswith('_factory.py') for name in file_list)
+        if not has_factory:
+            return ImportResult(
+                success=False,
+                plugin_name="",
+                message="无效的插件结构：缺少 *_factory.py 文件"
```

Ruff (0.15.13)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)